analyses/comparative-RNASeq-analysis/02-thresholds-and-outliers.py

```diff
@@ -0,0 +1,231 @@
+"""
+Generates gene outlier thresholds for normalized samples and uses them to
+calculate the outliers.
+
+Outlier thresholds are calculated according to a modified Tukey method, detailed
+below.
+
+Then, outliers are calculated based on these thresholds. In addition, for each sample,
+the 5% of genes with the highest expression are marked, as well as the genes which
+are excluded by the expression and variance filters from step 1.
+
+The final result is a matrix .rds file of genes by samples where, for each sample,
+each gene may be labeled with zero or more of the following keys (concatenated
+together into a string):
+
+U: Up outlier
+D: Down outlier
+T: in Top five percent of genes for this sample
```

Comment:
I assume this means top 5% in expression measure, not some other metric?

Reply:
Correct - updated doc.